dog-listing.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+<body>
+   <header>  
+   <img class="svg-dog-header" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+   <p>Seattle's premiere dog adoption agency</p>
+   <div>
+     <img class="svg-cart" src="images/cart-icon.svg" alt="checkout cart icon">
+     <input class="cart-btn" type="submit" value="Checkout">
+   </div>
+   </header>
+ 
+   <main>
+      <nav class="nav-head">
+         <a class="nav-item-head" href="index.html">Home<span class="space">/</span></a>
+         <a class="nav-item-head" href="dog-listing.html">Dogs<span class="space">/</span></a>
+         <a class="nav-item-head" href="contact.html">Contact</a>
+     </nav>
+     <section>
+        <h2>Here are all the Dogs!</h2>
+        <img src="images/murphy-card.jpg" alt="A picture of dog murphy">
```

Comment:
You'll want to wrap these dogs in some sort of container and use flex or grid to lay them out.